snappy/click.go

```diff
@@ -133,7 +133,7 @@ ubuntu-core-launcher {{.UdevAppName}} {{.AaProfile}} {{.Target}} "$@"
 	var templateOut bytes.Buffer
 	t := template.Must(template.New("wrapper").Parse(wrapperTemplate))
 	wrapperData := struct {
-		AppName     string
+		SnapName    string
 		AppArch     string
 		AppPath     string
```

Comment:
It really is the snap name here. It gets shoved into snapenv.GetDeprecatedBasicSnappyEnvVars and the like and this shows up as the related template change there.